quickstarts/quarkus-with-helm.yaml

```diff
@@ -0,0 +1,162 @@
+        
+        1. Click **Install Helm Chart**
+        
+            - The form includes multiple sections which are collapsed by default.  Browse these sections to get an idea of the available options, although no changes are needed to proceed. 
+        
+        1. At the bottom of the form, click **Install** to create the Quarkus application via the Helm Chart.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |-
+          The Helm Release is represented by the light grey shape with a darker grey dashed border.  The deployment is a white circle containing the Quarkus logo.  To verify the application was successfully created:
+          
+          - Do you see a **quarkus** Helm Release?
+          
+          - Do you see a **quarkus** deployment?
+
+      summary:
+        failed: Try the steps again.
+        success: Your Quarkus application has been deployed onto OpenShift.
+      title: Create a Quarkus application from a Helm Chart
+
+    - description: >-
+        To view the build status of the Quarkus application:
+        
+        1. Hover over the icon on the bottom left quadrant of the **quarkus** deployment to see the build status in a tooltip.
+        
+        1. Click on the icon for quick access to the build log.  
+        
+            - The log stream of the **quarkus-1** build is shown on the **Build Details** page.
+        
+        1. The application and its dependencies will be built into a container image and pushed to the OpenShift container registry.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: >-
+          To verify the build is complete:
+
+          - Wait for the build to complete.  It may take a few minutes.  Do you see a **Complete** badge on the page header?
+
+      summary:
+        failed: Try the steps again.
+        success: Your build is complete.
+      title: View the build status
+    - description: >-
+        To view the pod status:
+
+        1. Click the browser tab running OpenShift.  
+        
+            - Notice that the
+        **quarkus** deployment has a pod donut imposed on the
+        circle, representing the pod status (i.e. blue = running).  
+            - The color of the donut indicates
+        the pod status.  
+
+        1. Hover over the pod donut to see the pod status in a tooltip.
+
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again.
+        instructions: |-
+          To verify you see the pod status:
+          
+          - Do you see the number of associated pods and their statuses?
+
+      summary:
+        failed: Try the steps again.
+        success: Your deployment has one running pod.
+      title: View the pod status
+    - description: >-
+        Now let's Open the Quarkus application URL!  
```

Comment:
Open --> open ?

Reply:
I'm going to change this sentence across the board.